drizzlepac/wcs_functions.py

```diff
@@ -1074,10 +1074,17 @@ def make_mosaic_wcs(filenames, rot=None, scale=None):
 
     # Compile list of WCSs for all chips from all input filenames
     hstwcs_list = []
+    
     for f in filenames:
-        hstwcs_list.extend([get_hstwcs(f, extnum) for extnum in get_extns(f)])
-    # Combine them into a single mosaic WCS
-    output_wcs = utils.output_wcs(hstwcs_list, undistort=True)
+        if ('flc.fits' in f) | ('flt.fits' in f):
```

Comment:
`then` and `else` branches do the same thing. Why not combine this in the `if` clause:
```suggestion
        if ('flc.fits' in f) or ('flt.fits' in f) or ('drc.fits' in f) or ('drz.fits' in f):
```
(also, why bitwise operator instead of the logical one?)